### PR TITLE
	modified:   content/cpp (abs-acos-asin)

### PR DESCRIPTION
**Pull request by l2eza**
modified:   content/cpp/concepts/math-functions/terms/abs/abs.md
	modified:   content/cpp/concepts/math-functions/terms/acos/acos.md
	modified:   content/cpp/concepts/math-functions/terms/asin/asin.md

###	modified:   content/cpp/concepts/math-functions/terms/asin/asin.md

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by l2eza** [comment]
@SSwiniarski  thank you! 
I have one quetion: so I had a folder with edited .js codebytes. I opened a new branch in VS code for editing cpp codebytes, but the .js edits were still there. So I deleted the old folder and cloned it again in the new location -> new branch -> edits. Could you please let me know if this is right and should I do it every time? 🤔

**Comment by SSwiniarski** [comment]
@l2eza you should be able to checkout main again and create a new branch from there. And yo should do a new branch for each PR.

File: content/cpp/concepts/math-functions/terms/abs/abs.md [hunk 1]
```diff
@@ -32,3 +32,24 @@ int main() {
   // Output: 6.5
 }
 ```
+## Codebyte Example
```
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/abs/abs.md:35]:
```suggestion

## Codebyte Example
```

File: content/cpp/concepts/math-functions/terms/acos/acos.md [hunk 2]
```diff
@@ -37,3 +37,27 @@ int main() {
   // Output: 1.5708 radians
 }
 ```
+## Codebyte Example
```
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/acos/acos.md:40]:
```suggestion

## Codebyte Example
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@l2eza this looks clean. Just a few comments adding a line break before the headers.

**Comment by l2eza** [comment]
@SSwiniarski ah awesome! thank you 🙌

**Comment by l2eza** [comment]
> @l2eza you should be able to checkout main again and create a new branch from there. And you should do a new branch for each PR.

Got it! 🙏

**Comment by yangc95** [comment]
Hey @l2eza, this is Christine from the Curriculum Free Team. First of all, thank you for taking on this entry and opening a PR for Codecademy Docs. However, it looks like the codebytes are identical to the code used in:

https://www.programiz.com/cpp-programming/library-function/cmath/abs (example2)
https://www.programiz.com/cpp-programming/library-function/cmath/asin (example1)

I'm going to have to close this PR and ask that you open a new PR with original examples. For reference, here is Codecademy's statement on plagiarism:

> At Codecademy, you are expected to create original content. It is your responsibility to avoid plagiarism: the practice of taking someone else's work or ideas and presenting them as your own.
>
> Plagiarism can include:
>
> - Copying a phrase, sentence, or passage from another writer without proper citation.
> 
> - Incorporating a piece of text while making small changes to the original.
> 
> - Copying large amounts of text with or without citation.
> 
> - Reusing your own writing from a non-Codecademy source.
> 
> - Copying sections of source code from another programmer as your own.
> 
> - Reusing your own code from a non-Codecademy source.
> 
> - Using external resources (images, videos, datasets, etc.) without proper citation.
>
> All written content produced for Codecademy should be free of plagiarism. This includes drafts of content items as well as published content.

Please let me know if you have any questions regarding this or with rewriting the entry. 😄 

As you rewrite this entry and open a new PR, please refer to the following resources for guidance:

- [Codecademy Docs Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md)
- [University of Oxford article on plagiarism](https://www.ox.ac.uk/students/academic/guidance/skills/plagiarism)

If you have any open PRs that include any plagiarized content, please take the opportunity to revise.

**Pull request closed without merging**